satellite/console/consoleweb/server.go

```diff
@@ -206,14 +206,16 @@ func (server *Server) appHandler(w http.ResponseWriter, r *http.Request) {
 
 	cspValues := []string{
 		"default-src 'self'",
-		"script-src 'self' *.stripe.com cdn.segment.com",
+		"frame-ancestors tardigrade.io",
```

Comment:
why should storj repo knows smth about tardigrade branding? Would it better to place this values in config?

Reply:
actually, the same with segment and stripe...

Reply:
I'll handle my addition. Those can be configured later.